services/data-info/src/data_info/routes/avus.clj

```diff
@@ -13,25 +14,30 @@
     (GET* "/avus" [:as {uri :uri}]
       :query [{:keys [user]} StandardUserQueryParams]
       :return AVUGetResult
+      :middlewares [wrap-metadata-base-url]
```

Comment:
This doesn't appear to work, at least not with the version of compojure-api we're using.
We should revisit this once we update to the latest version of compojure-api.